services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -302,11 +303,22 @@ async def list_projects(
                     )
                     & (
                         (projects_to_products.c.product_name == product_name)
+                        # This was added for backward compatibility, including old projects not in the projects_to_products table.
                         | (projects_to_products.c.product_name.is_(None))
                     )
                 )
-                .order_by(desc(projects.c.last_change_date), projects.c.id)
             )
+            if search:
```

Comment:
what if search="" ?

Reply:
Thanks added validator and test for this use case -> it will behave as it is None and the search query is not provided